docs/ext/thread-pools.md

```diff
@@ -0,0 +1,65 @@
+# Thread Pools
+
+Java's [ThreadPoolExecutor](http://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadPoolExecutor.html)
+exposes several properties that are useful to monitor to assess the health, performance, and configuration of the pool.
+
+## Getting Started
+
+To report thread pool metrics, one can attach a
+[ThreadPoolMonitor](http://netflix.github.io/spectator/en/latest/javadoc/spectator-api/com/netflix/spectator/api/patterns/ThreadPoolMonitor.html)
+in the following manner:
```

Comment:
Consider using reference style links to avoid having the long URIs in the middle of the paragraph when reading the raw markdown.

https://daringfireball.net/projects/markdown/syntax#link

Reply:
Thanks; wasn't aware of that.  I've only used very simple markdown to date.